lib/html_helper.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+require 'sinatra/base'
```

Comment:
Why is this required? I can't see how this helper is really connected with Sinatra at all, and if I remove this the tests all pass, and the Bahamas page still seems to do the right thing.